package.json

```diff
@@ -8,21 +8,24 @@
     "node": ">=6.13.0"
   },
   "scripts": {
-    "reset-db": "mysql -u root < schema.sql",
+    "reset-db": "mysql -h 'db' -u root < schema.sql",
```

Comment:
nice use of a flexible IP from docker compose

Reply:
thanks